custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -66,6 +101,10 @@ def _config_logger(options):
         logger.setLevel(LOG_LEVELS[options.logging_level])
         return logger
 
+    def _set_target_type(self, target_type):
+        self.target_type = target_type
+        self.options.target_type = target_type.value
+
```

Comment:
I'm not a fan of this information being stored twice.

Reply:
the best would be to save model related params in a separate structure, like context, instead of passing options everywhere.

Reply:
done